contribs/noise/src/main/java/org/matsim/contrib/noise/NoiseReceiverPoint.java

```diff
@@ -224,11 +224,17 @@ void resetTimeInterval() {
 	}
 
 	void processImmission() {
+		// yy why is it meaningful to not compute them in the getters?  (If the getters are called very often, then precomputing the relatively expensive
```

Comment:
thanks for pointing that out. we can actually save three fields for each receiver point in that case. I will update accordingly